lib/telemetry_metrics_statsd.ex

```diff
@@ -234,7 +234,7 @@ defmodule TelemetryMetricsStatsd do
   @type prefix :: String.t() | nil
   @type option ::
           {:port, :inet.port_number()}
-          | {:host, String.t()}
+          | {:host, String.t() | :inet.ip_address()}
```

Comment:
Could you make a new `host()` type and use it here and in the typespec for `open/2`?

Reply:
Done